docs/technical_reference/unit_models/uv_aop.rst

```diff
@@ -98,16 +98,30 @@ When setting the `uv_dose_type` configuration option to `UVDoseType.calculated`,
    "Relative lamp output", ":math:`\frac{S}{S_0}`", "relative_lamp_output", None, None
    "Number of banks", ":math:`N_{bank}`", "num_of_banks", None, None
 
+When setting the ``has_aop`` configuration option to ``True``, there are 2 additional variables that must be fixed.
+This leads to a total of 7 degrees of freedom. Additional variables that must be fixed include:
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
+
+   "Pseudo-first order rate constant for direct photolysis of component", ":math:`k_d`", "photolysis_rate_constant", "[p, j]", ":math:`\text{s}^{-1}`"
+   "Pseudo-first order rate constant for indirect photolysis of component", ":math:`k_i`", "reaction_rate_constant", "[p, j]", ":math:`\text{s}^{-1}`"
+   "Second-order reaction rate constant", ":math:`k_{OH}`", "second_order_reaction_rate_constant", "[p, j]", ":math:`\text{M}^{-1} \text{s}^{-1}`"
+   "Steady-state concentration of hydrogen peroxide", ":math:`c_{\text{H}_2\text{O}_2}`", "hydrogen_peroxide_conc", None, ":math:`\text{M}`"
+
+**Users must provide values for and 'fix' certain variables to solve the model with DOF=0. Thus, users should fix**
+    * either 'photolysis_rate_constant' or 'reaction_rate_constant',
```

Comment:
Firstly, this should now be removed from line 80 I believe since it is now only constructed when `has_aop=True`.

Secondly, would it make sense that when `has_aop=False`, `rate_constant` should be fixed. When `has_aop=True`, `rate_constant` should be unfixed and both `photolysis_rate_constant `or `reaction_rate_constant `should be fixed. How are these parameters obtained, and does this make sense?

Reply:
I'm on the same page when `has_aop=False` that "the `rate_constant` can be derived from `inactivation_rate` and `uv_intensity` or directly fixed. It also seems like the `inactivation_rate` is the governing variable in the mass transfer equation so this would be the direct specification.

I think that the `rate_constant` in this equation is fundamentally the `photolysis_rate_constant` though when `has_aop=False` and that is where there is some confusion when `has_aop=True`. Maybe having the `reaction_rate_constant.fix(0)` when `has_aop=False` is more clear, but I think you may of mentioned you used to have it this way? If this is true, then `rate_constant` which is the `overall_rate_constant` should never be fixed as only the `photolysis_rate_constant` or `reaction_rate_constant` are experimentally derived. 

Hopefully I am interpreting this correctly.

Reply:
So given the equations,
```
b.rate_constant[p, j] == b.uv_intensity * b.inactivation_rate[p, j]
b.rate_constant[p, j] == b.photolysis_rate_constant[p, j] + b.reaction_rate_constant[p, j]
```

Basically if for a theoretical system that,
```
when has_aop=False
     b.rate_constant[p, j]=10
```
I would suspect that b.photolysis_rate_constant[p, j]=10 and b.reaction_rate_constant[p, j]=0

Then for the same theoretical system and introducing aop, if you keep the `b.rate_constant[p, j]=10` specification, there is no mathematical way it exists and instead should be,
```
when has_aop=True
    b.photolysis_rate_constant[p, j]=10
    b.reaction_rate_constant[p, j]=5
```

Reply:
I finally decide to take your suggestion. Right now, both `photolysis_rate_constant` and `reaction_rate_constant` are default variables and `reaction_rate_constant` is fixed to 0 when `has_aop=False`.